portfolio/src/main/webapp/style.css

```diff
@@ -4,13 +4,23 @@
   width: 80%;
 }
 
+#about-me {
+  margin-left: auto;
```

Comment:
[Prefer using the `margin` shorthand](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)